web-ui/src/game-engine/gameStateModifiers.ts

```diff
@@ -39,33 +38,21 @@ export class GameStateModifier {
   };
 
   setLegalBiddingMoves = (legalMoves: LegalBiddingMove[]) => {
-    if (this.gameState.currentPhase !== GameRoundPhase.BIDDING) {
```

Comment:
We can receive the next legal bidding moves just before receiving the fact that the bidding phase actually started...